AstSemantics.md

```diff
@@ -111,6 +112,9 @@ Integer loads can specify a *storage size* which is smaller than the result type
 well as a signedness which determines whether the bytes are sign- or zero-
 extended into the result type.
 
+Each `load` and `store` specifies the [linear memory index](Modules.md#index-spaces)
+to operate on with an unsigned integer immediate.
+
```

Comment:
@jfbastien The 'code sharing concern' I noted is that it is not possible to both patch or baking-in constant offsets to module linear memory bases, while also sharing the same code between different instances. I don't think the heap indexes would be compiled into indirections through a table of heap bases at runtime, which would be very costly, rather into access offsets?